nestkernel/event.h

```diff
@@ -827,13 +843,20 @@ class GapJEvent : public SecondaryEvent
   static void
   set_syn_id( const synindex synid )
```

Comment:
The compiler just "reminded" me that we cannot add `assert( not supports_syn_id( synid ) );` to `set_...` because the call of `supports_syn_id` requires an object of the class. And we also cannot make `supports_syn_id` static as it is also a virtual function of the base class.

So I guess we will have to live without an `assert` here